cli/popper/commands/cmd_run.py

```diff
@@ -96,6 +96,39 @@ def cli(ctx, pipeline, timeout, skip, ignore_errors, output,
         pu.fail("Failed to execute pipeline")
 
 
+def check_for_env_vars(args):
+    env_vars = args[-1]
+    if env_vars:
+        return run_env_vars(args)
+    else:
+        return run_pipeline(*args[:-1])
+
+
+def run_env_vars(args):
```

Comment:
if the above gets implemented, then we don't need a specific case for `run_env_vars` vs. `run_pipeline`, in other words, the logic of handling a list of executions will be part of the `run_pipeline` function.